package.json

```diff
@@ -22,21 +22,22 @@
     "less-loader": "4.1.0",
     "mini-css-extract-plugin": "0.4.5",
     "moment-locales-webpack-plugin": "1.1.0",
+    "moment-timezone-data-webpack-plugin": "^1.1.0",
     "optimize-css-assets-webpack-plugin": "5.0.1",
     "postcss-loader": "3.0.0",
     "postcss-safe-parser": "4.0.1",
     "purify-css": "1.2.5",
     "purifycss-webpack": "0.7.0",
     "terser-webpack-plugin": "1.2.1",
-    "webpack": "4.28.3",
+    "webpack": "^4.42.1",
```

Comment:
Is this deliberately set to auto-update?

Reply:
Yes, it doesn't matter if a newer 4.x version is used as long as it gets a new enough version to have the logging API that's used